wagtail/admin/templates/wagtailadmin/skeleton.html

```diff
@@ -19,6 +19,54 @@
     {% block branding_favicon %}{% endblock %}
 </head>
 <body id="wagtail" class="{% block bodyclass %}{% endblock %} {% if messages %}has-messages{% endif %} focus-outline-on">
+    <div data-sprite></div>
+    <script>
+        function loadIconSprite() {
+            var spriteURL = '{% url "wagtailadmin_sprite" %}';
+            var revisionKey = 'spriteRevision';
+            var dataKey = 'spriteData';
```

Comment:
I could see this potentially colliding with Wagtail sites’ code since Wagtail is generally served from the same domain as sites (at least in development). I think it would be worth prefixing both of those keys with `wagtail:` or similar, so they are less likely to be un-intentionally shared?

I could see myself stealing this icon sprites implementation for other Wagtail-based projects 😉 .

```suggestion
            var revisionKey = 'wagtail:spriteRevision';
            var dataKey = 'wagtail:spriteData';
```